src/containers/TasksContainer.test.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Created by Katarzyna_Bak on 31.05.2017.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import TasksContainer from './TasksContainer';
+import configureStore from 'redux-mock-store';
+import { addTask, doneTask } from '../actions';
+
+
+describe('[Container] TaskContainer)', () => {
```

Comment:
In general, testing `mapStateToProps` and `mapDispatchToProps` functions is enough to test containers.